publify_core/app/views/admin/content/index.js.erb

```diff
@@ -1,2 +1,2 @@
 $("#articleList").html('<%= raw escape_javascript(render(partial: "article_list", locals: { articles: @articles })) %>');
-$("#spinner").show();
+$("#spinner").hide();
```

Comment:
This works, but there is a more elegant solution. 

There is already something in the codebase that is supposed to run after the search `ajax` call completes. However, it is slightly broken. I would find where that code is and fix it, instead of adding new code in a separate file.

Reply:
Actually, once you make this more elegant fix, you could get rid of this line in this file.